db/migration/000002_cli_user.up.sql

```diff
@@ -0,0 +1,25 @@
+
+ALTER TABLE users ADD COLUMN created_by UUID;
+-- SQL Definition for environments
+
+CREATE TABLE environments (
+    Id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    cli_user UUID NOT NULL REFERENCES users(id),
```

Comment:
I am assuming that this ties an environment to a single user

Reply:
yes